src/components/AddBookForm.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components/macro'
+
+import Button from './RectangleButton'
+import downArrow from '../assets/arrow-down.svg'
```

Comment:
You could import your svgs like this: `import {ReactComponent as ReactLogo} from './logo.svg';`. It'll then render as svg in your code and you don't need the img element.
```suggestion
import { ReactComponent as DownArrow } from '../assets/arrow-down.svg'
```